infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -690,16 +645,10 @@ public Layout copyManagementLayout() {
                 .setNameFormat(getThreadPrefix() + "client-%d")
                 .build();
 
-        final int requestedThreads =
-            Integer.parseInt(getServerConfig(String.class, "--Threads"));
-        final int numThreads = requestedThreads == 0
-            ? Runtime.getRuntime().availableProcessors() * 2
-            : requestedThreads;
-        EventLoopGroup group = getChannelImplementation().getGenerator()
-            .generate(numThreads, threadFactory);
+        EventLoopGroup group = configuration.getChannelImplementation().getGenerator().generate(6, threadFactory);
 
         log.info("getClientGroup: Type {} with {} threads",
-            group.getClass().getSimpleName(), numThreads);
+            group.getClass().getSimpleName(), 6);
```

Comment:
@Maithem this hardcode was in the previous commit, will revert to master during cleanup